ntbs-service/wwwroot/css/notificationBanner.scss

```diff
@@ -0,0 +1,83 @@
+@import "./_colors.scss";
+
+.notification-banner-head-container {
+    color: white;
+    padding: 15px 20px;
+}
+
+.notification-banner-title-text {
+    font-weight: 600;
+    font-size: x-large;
+    margin-bottom: 0;
+}
+
+.notification-banner-body-container {
+    padding: 15px 20px;
+}
+
+.notification-banner-field {
+    padding-bottom: 10px;
+}
+
+.notification-banner-links-container {
+    padding: 5px 10px;
+    border-bottom: 1px solid $draft-grey;
+    margin-bottom: 20px;
+    display: flex;
+}
+
+.notification-banner-link-separator {
+    margin: 2px 10px;
+    border-right: 2px solid black;
+}
+
+.notification-banner-triangle {
+    position: absolute;
+    width: 0; 
+    height: 0; 
+    border-left: 20px solid transparent;
+    border-right: 20px solid transparent;
+    right: 40px;
+}
+
+.notification-banner--draft {
+    border: 2px solid $draft-grey;
+    position: relative;

```

Comment:
Should we have a `notification-banner` class that contains the `position` and `margin-bottom` properties? Then we don't have to duplicate them between the `--draft` and `--notified` classes

Reply:
Committed this during training and didn't mean to push, used to upsource! 😰 but as a direct reply to this comment happy to change - I've had different people prefer both styles when there are only 1-2 other styling properties but think I'd prefer to go down your route so will change :)